pkgs/data/themes/tokyo-night-gtk/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, gnome-themes-extra
```

Comment:
You should also remove the now unused param. And please squash the fixup commits.

Reply:
done